theme/forms.py

```diff
@@ -225,6 +226,12 @@ class Meta:
         model = User
         exclude = ['last_login', 'date_joined', 'password']
 
+    # redefining the Django username model field in order to change the invalid username message
+    # original message indicates that / is a valid character
+    username = forms.CharField(validators=[validators.RegexValidator(r'^[\w.@+-]+$',
```

Comment:
@pkdash Note that Django username rule is a bit different between version 1.9 and 1.10. I am wondering whether there is a way to directly catch Django user name exception if the user name violates Django user name rules rather than we write our own regex validator trying to conform to Django username rules which could change from version to version, which we may not be able to keep up with over time.